agent/exec/containerd/executor.go

```diff
@@ -0,0 +1,107 @@
+package containerd
```

Comment:
I wouldn't mind at all...

We should consider whether `ContainerSpec` should become something more `dockerapi`ish too.